2048/2048/src/main/java/com/appcoins/eskills2048/MainGame.java

```diff
@@ -62,6 +68,7 @@
   private boolean playing = true;
   private static final int MAX_CHAR_DISPLAY_USERNAME = 11;
   private final UserDetailsHelper userDetailsHelper;
+  private boolean exceptionEndedGamge = false;
```

Comment:
Why are you creating this global variable when you can simply pass it as method argument?

Reply:
Agree